app.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
```

Comment:
- [ ] May you want to say "students"? Please 🙏🏽 correct this line to avoid issues 👇🏽 Also check the classroom.rb file for the same error.

![image](https://github.com/hasnatali1947/-OOP_school_library/assets/58642949/65abec94-7a2a-4f44-a84a-3f80b15a4984)